### ISSUE TITLE
Gutenboarding: Introduce category descriptions (and starter design descriptions for screen readers)

### ISSUE THREAD

**Issue by ianstewart** [Automattic/wp-calypso#45038]
It's been proposed that we add "categorization" to our design screen in Gutenboarding. This issue will stub out a first pass at a set of category descriptions.

Additionally, it's been proposed that we add meta descriptions for each design for screen readers. Those same category descriptions could be used for a first pass at design descriptions as well. For now. (Have to start somewhere.)

Here, in text, to start are a short list of categories that we can assign starter designs to. (Will need a mockup later and that'll be added so we have a _visual_ of what goes where but I wanted to jot this down first.) I think the first description I stole from @olaolusoga.

Blog
Create a space on the web where you can document important content that expresses who you are.

Portfolio
Share your visual work with the world with a design that displays your imagery and photos in artful galleries.

Services
Tell the world about the local or online services you offer with a simple site in a professional design.

Restaurants
Share your menu, add your hours, and location, and let the whole world know about takeout and delivery options for your restaurant with a Restaurant site

cc @ollierozdarz @rickybanister @alaczek

**Comment by alaczek** [comment]
I took a stab at categorizing designs currently available in signup:

**Blog**
Bowen, Edison, Cassel

**Portfolio**
Vesta, Easly, Camden, Reynolds, Mayland, (Coutoire  - I'm not 100% sure about it)

**Small Business**
Doyle, Overton, Brice, Barnsbury, Alves, Rivington, Balasana, Stratford, Leven

**Restaurant**
Maywood, Rockfield, Overton

Here's how this looks like (hat tip to @ianstewart)

![Starter Design Categories](https://user-images.githubusercontent.com/4550351/90996373-f18b1080-e601-11ea-9fc0-c850a9fed965.jpg)

Things are not balanced with Blog and Restaurant categories having only three designs, while Small Business has nine. This is mostly because we added a bunch of Varia child themes to signup, which are predominantly geared towards small businesses. We have a bunch of designing in the works that should help populate the blog category at the very least.

**Comment by rickybanister** [comment]
Just for conjecture's sake, could restaurant be folded into services or the more generic 'business' or something like that? It'll be clear to customers that they're for restaurants and it does feel a bit niche to have its own category.

**Comment by simison** [comment]
Perhaps good first steps would be to...

In design storage (no UI yet):
- Add categories to different designs
- Add simple descriptions for each design, like "Dark colored design with big header image."

In design grid UI, generate ARIA descriptions for screen-readers from the above info:
```html
aria-description="Dark colored design with big header image. Good for business sites."
```

All of the above could be good developer-onboarding material to get done soon (cc @andrewserong since this touches Ganon's work on these APIs?)

**Comment by andrewserong** [comment]
For the Patterns API, I imagine a good place to source _individual_ descriptions for each design would be to use the post excerpt field in the editor when we're building the designs? In the short-term, it'd make sense to add designs descriptions / category descriptions to the `available-designs-config.json` file.

**Comment by andrewserong** [comment]
Adding Patterns API support for a description field sourced from `post_excerpt` in D48611-code.

**Comment by ollierozdarz** [comment]
@ianstewart @olaolusoga @simison we've chatted about getting categories into Gutenboarding at some time or another over the past months. The reminder above seems like a good time to just try to make it happen. In my mind, the implementation (at least to start off) would just be a short and punchy list which wouldn't require descriptions.

**Comment by simison** [comment]
Yep, I think we can make this happen with small effort. I believe there even was design already, or we can reuse the page template picker design? @ollierozdarz 

What categories would we have? I think listing them is an useful excercise anyway, just to understand better what we have right now.

**Comment by ollierozdarz** [comment]
@simison, yep we've got a few simple designs to choose from (depending on if we use one or two column layout) which I can surface.

> What categories would we have? I think listing them is an useful excercise anyway, just to understand better what we have right now.

I think the Themeshaper (@ianstewart) himself is our best bet at accurately answering this 🙂

**Comment by ianstewart** [comment]
I think we're not too far off from what's above from @alaczek …

https://github.com/Automattic/wp-calypso/issues/45038#issuecomment-678866574

**Comment by ollierozdarz** [comment]
Cool. Thanks, @ianstewart!

**Comment by alshakero** [comment]
Here is my old attempt on design picker categories https://github.com/Automattic/wp-calypso/pull/45091

**Comment by razvanpapadopol** [comment]
@alaczek @ianstewart is there an updated list of categories we want to be displayed for filtering designs or https://github.com/Automattic/wp-calypso/issues/45038#issuecomment-678866574 shows the most recent list? 

We've seen in the [designs config file](https://github.com/Automattic/wp-calypso/blob/trunk/packages/design-picker/src/available-designs-config.json) some other categories as well. Eg: 'small business' or 'charity' or 'non-profit'.

**Comment by simison** [comment]
Just heads up that Ian is AFK this week. :-)

I don't think those categories in the design config file were ever used and were just for an example initially, but I could be wrong, it's been a while. :-)

**Comment by ollierozdarz** [comment]
👋 @simison @razvanpapadopol, could we use @alaczek's suggestions ([#45038 (comment)](https://github.com/Automattic/wp-calypso/issues/45038#issuecomment-678866574)) for a first pass? I think there might only be one or two newly introduced designs which are missing.

In terms of implementation, I'm thinking either of these basic layouts (we'll keep the existing onboarding styles though, just showing the 2-col layout vs single-col):

<img width="1438" alt="Screen Shot 2021-04-06 at 9 22 21 am" src="https://user-images.githubusercontent.com/67085494/113638253-c4cfc900-96b9-11eb-8160-73b976b9b506.png">
<img width="1440" alt="Screen Shot 2021-04-06 at 9 22 10 am" src="https://user-images.githubusercontent.com/67085494/113638258-c7322300-96b9-11eb-83ef-35184e14e1af.png">

In terms of initial implementation, I'm thinking the categories above the designs will be the least disruptive to the page?

**Comment by autumnfjeld** [comment]
@ianstewart @alaczek  Can this issue be closed?
